pulp_integrity/pulp_rpm_integrity/monkey.py

```diff
@@ -0,0 +1,26 @@
+import os
+import types
+from pulp.plugins.loader import manager
+
+
+def rpm_monkey():
```

Comment:
older Pulp deployments might be missing `unit.get_symlink_name` but I'd like those to work anyway... See also: https://github.com/pulp/pulp/pull/3465#pullrequestreview-114935398